include/functions.h

```diff
@@ -2087,7 +2087,7 @@ void func_8010A33C(GlobalContext* globalCtx, s16 param_2);
 // void func_8010BEBC(void);
 // void func_8010BEF0(void);
 // void func_8010BF24(void);
-s32 func_8010BF58(Actor* actor, GlobalContext* globalCtx, s32* param_2, s32 param_3, s32* param_4);
+s32 func_8010BF58(Actor* actor, GlobalContext* globalCtx, s32* param_3, s32 param_4, s32* param_5);
```

Comment:
Weren't these right before?

Reply:
The convention I've seen is that generic `arg` arguments is 0-indexed but generic `param` arguments are 1 indexed. As much as I dislike params because of it's 1-index (looking at you matlab), the new change here would be consistent with param being a 1-index (`func_8010BF58(Actor* param_1, GlobalContext* param_2, s32* param_3)`)

Reply:
Oh yuck. I think we should eliminate that ASAP, then, that's just plain confusing.

Reply:
`arg` as 0-indexed makes sense to me as a direct parallel to the MIPS A0-A3 registers, but I did not realize we usually did `param` as 1-indexed. In that case, this change makes a lot of sense.

Reply:
Why are we even using two different things in the first place?

Reply:
Hence my preference for arg, but at the same time this is a problem that naturally goes away as documentation continues, since they're all temp placeholders anyway

Reply:
Looks like there are hardly any in code, but hundreds in functions.h... *sigh*